_episodes/01-numpy.md

```diff
@@ -60,21 +60,22 @@ it. In essence, **a variable is just a name for a value**.
 
 In Python, variable names:
 
- - must begin with a letter, and
+ - can include letters, digits, and underscores
+ - cannot start with a digit
  - are [case sensitive]({{ page.root }}/reference/#case-sensitive).
 
 This means that, for example:
  - `weight0` is a valid variable name, whereas `0weight` is not
  - `weight` and `Weight` are different variables
 
 ## Types of data
-Python knows various types of data. The most common ones are:
+Python knows various types of data. Three common ones are:
```

Comment:
common -> basic?

Reply:
I am not sure that "basic" is better, but I think either way works.